src/olympia/activity/utils.py

```diff
@@ -195,3 +218,20 @@ def filter_queryset_to_pending_replies(queryset, log_type_ids=NOT_PENDING_IDS):
     if not latest_reply:
         return queryset
     return queryset.filter(created__gt=latest_reply.created)
+
+
+def bounce_mail(message, reason):
+    if (not type(message) is dict or
+            ('From' not in message and 'ReplyTo' not in message)):
+        log.error('We needed a From or ReplyTo to bounce but didn\'t get one')
```

Comment:
We might have some email headers from the json, depending on how mangled it is.  I think we're best logging them further up wherever the exception occurred - the `log.error` here was more so we weren't silently existing from `bounce_mail` without any indication if an email was bounced or not.